src/foraging_gui/Foraging.py

```diff
@@ -1000,6 +1051,8 @@ def _GetSettings(self):
                 4:'D'
             }
             self.current_box='{}-{}'.format(self.current_box,mapper[self.box_number])
+        self.Other_current_box=self.current_box
+        self.Other_go_cue_decibel=self.Settings['go_cue_decibel_box'+str(self.box_number)]
```

Comment:
What is `self.Other_go_cue_decibel` for?

Reply:
Similar to Other_current_box, to automatically save the go cue decibel to the behavior json.